tests/common.go

```diff
@@ -228,23 +254,28 @@ func DeleteVolumesAndWait(ctx *scheduler.Context) {
 }
 
 // ScheduleAndValidate schedules and validates applications
-func ScheduleAndValidate(testname string) []*scheduler.Context {
+func ScheduleAndValidate(testname string, vps_map map[string]string) []*scheduler.Context {
```

Comment:
instead of using a map can we use ScheduleOptions as a parameter?

Reply:
Used the ScheduleOption to pass the vps related data, as a result , passing of this parameter is not required